src/apps/vpn/cmake/android.cmake

```diff
@@ -0,0 +1,86 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is just a copied file right? How annoying would it be to fix the git history so that we know it is moved and not written from scratch?

Reply:
yes, plus path changes -> src/platfroms/android -> src/apps/vpn/platforms/android